### PR TITLE
Add files removed earlier

### PR DESCRIPTION
**Pull request by jdfreder**
Mergeable version of #8500

Don't know why, but GitHub, despite me force pushing this same commit from two different machines, thinks the other PR has a merge conflict but not this one.

@takluyver

### CODE REVIEW

File: docs/source/conf.py [hunk 1]
```diff
@@ -193,7 +193,7 @@
 intersphinx_mapping = {'python': ('http://docs.python.org/2/', None),
                        'rpy2': ('http://rpy.sourceforge.net/rpy2/doc-2.4/html/', None),
                        'traitlets': ('http://traitlets.readthedocs.org/en/latest/', None),
-                       'jupyterclient': ('http://jupyter-client.readthedocs.org/en/latest/', None),
+                       'jupyter-client': ('http://jupyter-client.readthedocs.org/en/latest/', None),
```
Comment (takluyver) [inline docs/source/conf.py]: Please don't change this, there's already one reference using it as `jupyterclient`. Also, Sphinx gives a warning if it contains a dash, though it doesn't seem to break anything.

File: docs/source/install/kernel_install.rst [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+IPython can be installed (different python versions, virtualenv or conda 
+environments) as a kernel by following these steps:
+
+* make sure that the desired python installation is active (e.g. activate the environment)
+  and ipython is installed
+* run once ``ipython kernelspec install-self --user`` (or ``ipython2 ...`` or ``ipython3 ...``
+  if you want to install specific python versions)
+
+The last command installs a :ref:`kernel spec <jupyter-client:kernel-specs>` file for the current python installation. Kernel spec files are JSON files, which can be viewed and changed with a
```
Comment (takluyver) [inline docs/source/install/kernel_install.rst]: The ref in jupyterclient is `kernelspecs` (no hyphen)

File: docs/source/conf.py (hunk 1 above)
Comment (jdfreder) [inline docs/source/conf.py]: Grumble, must be a rebase error

**Comment by jdfreder** [comment]
@takluyver comments addressed, and commits squashed.

**Comment by takluyver** [comment]
Thanks Jon

**Comment by jdfreder** [comment]
Thank you, for your patience and continuous error checking. : )

**Comment by takluyver** [comment]
ThomasCI ;-)

**Comment by jdfreder** [comment]
ThomasCI > TravisCI

**Pull request merged**